choosing.md

```diff
@@ -19,7 +19,7 @@ applications of environments:
 - A disposable environment for testing a new library without polluting the rest
   of the Python install; this environment can easily be discarded when no
   longer needed.
-- Quick set up of a known set of dependencies for workshops and tutorials
+- Quick set up of a known set of dependencies for workshops and tutorials.
```

Comment:
This is not currently a sentence, so no period. That also goes for the one above.